src/modules/orders/templates/sms/sms.template.ts

```
@@ -0,0 +1,8 @@
+export function orderStatusSMSTemplate(phoneNumber: string, orderId: string, status: string) {
```

ESLint

[error] Replace `phoneNumber:·string,·orderId:·string,·status:·string` with `⏎··phoneNumber:·string,⏎··orderId:·string,⏎··status:·string,⏎`

(prettier/prettier)